hooks/go/060-migrate-from-ceph-csi-module/migrate-from-ceph-csi-module.go

```diff
@@ -160,10 +160,11 @@ func handlerMigrateFromCephCsiModule(ctx context.Context, _ *pkg.HookInput) erro
 			cephClusterConnection.Name = cephClusterConnectionName
 			cephClusterConnection.Labels = map[string]string{CephCSIMigratedLabel: CephCSIMigratedLabelValueTrue}
 			cephClusterConnection.Spec = v1alpha1.CephClusterConnectionSpec{
-				Monitors:  cephCSIDriver.Spec.Monitors,
-				ClusterID: cephCSIDriver.Spec.ClusterID,
-				UserID:    cephCSIDriver.Spec.UserID,
-				UserKey:   cephCSIDriver.Spec.UserKey}
+				Monitors:       cephCSIDriver.Spec.Monitors,
+				ClusterID:      cephCSIDriver.Spec.ClusterID,
+				UserID:         cephCSIDriver.Spec.UserID,
+				UserKey:        cephCSIDriver.Spec.UserKey,
+				SubvolumeGroup: cephCSIDriver.Spec.CephFS.SubvolumeGroup}
```

Comment:
Check that `cephCSIDriver.Spec.CephFS` exists and that `cephCSIDriver.Spec.CephFS.SubvolumeGroup` is not empty.